tests/UKHO.S100PermitService.API.FunctionalTests/Helpers/TestConfiguration.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Microsoft.Extensions.Configuration;
+
+namespace UKHO.S100PermitService.API.FunctionalTests.Helpers
+{
+    public class TestConfiguration
+    {
+        protected IConfigurationRoot configurationRoot;
```

Comment:
Please use the underscore convention for non-public instance members, so _configurationRoot. Could you adjust your editor configuration accordingly?

Reply:
I have made the required change.